src/main/java/com/yelp/nrtsearch/server/grpc/LuceneServer.java

```diff
@@ -81,8 +83,6 @@
 import java.util.concurrent.Future;
 import java.util.concurrent.ThreadPoolExecutor;
 import java.util.stream.Collectors;
-import me.dinowernli.grpc.prometheus.Configuration;
```

Comment:
Hmm, this is unfortunate that we basically cannot extend `me.dinowernli`.  I can think of following alternatives

1. Use [raw prometheus simpleclient](https://github.com/prometheus/client_java#instrumenting) which this library was doing anyway. Might be more re-work though.

2. Maybe use [asm](https://dzone.com/articles/fully-dynamic-classes-with-asm) to generate dynamic class that could extend ServerMetrics 

3. use your implementation here as is but since we are pretty much rolling out our own implementation of `me.dinowernli`  should we remove that dep from the build.gradle? You will probably also need to fix the tests using it in  `GrpcServer`

As long as this allows us to easily add new metrics going forward I am okay with the current approach you take.